sagemaker-training-compiler/tensorflow/single_gpu_single_node/hyper-parameter-tuning.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>For the link from <code>[the documentation]()</code>, there is empty parens instead of a link provided. Also please use the page title for the link text.</p><p> </p><p>For the link from "reuse and restart a tuning job with an extended range," use the doc title - suggested: <code>You can always reuse and restart a tuning job with an extended range, as explained in [Run a Warm Start Hyperparameter Tuning Job](https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-warm-start.html)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>